src/main/java/baritone/process/BuilderProcess.java

```diff
@@ -911,7 +912,8 @@ public double costOfPlacingAt(int x, int y, int z, IBlockState current) {
 
         @Override
         public double breakCostMultiplierAt(int x, int y, int z, IBlockState current) {
-            if (!allowBreak || isPossiblyProtected(x, y, z)) {
+//            baritone.getPathingBehavior().updateIsCloseToHome();
+            if (!allowBreak || isPossiblyProtected(x, y, z) || baritone.getPathingBehavior().isCloseToHome()) {
```

Comment:
and another cache-or-per-block

and why is pathing disallowed to place or break near home waypoints, while the builder may place but not break there? That's a bit inconsistent

Reply:
What? I thought I builder couldn't place either...

Reply:
Ahhhhhh, BuilderProcess uses CalculationContext's hasThrowaway, I just searched for allowPlace... -_-

Reply:
Changed in latest commit